rainbows.rb

```diff
@@ -17,6 +19,8 @@
   [255,0,125] => "raspberry"
 }
 
+rainbow = [{red:{r:255,g:0,b:0}}, {orange:{r:255,g:127,b:0}}, {yellow:{r:255,g:255,b:0}}, {green:{r:0,g:255,b:0}}, {blue:{r:0,g:0,b:255}}, {violet:{r:75,g:0,b:130}}, {ultra_violet:{r:143,g:0,b:255}}]
+
```

Comment:
Each hash for the colors should be on separate lines for readability and ease in commenting out, etc.